staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -71,7 +71,8 @@ func DeleteResource(r rest.GracefulDeleter, allowsOptions bool, scope *RequestSc
 
 		options := &metav1.DeleteOptions{}
 		if allowsOptions {
-			body, err := limitedReadBodyWithRecordMetric(ctx, req, scope.MaxRequestBodyBytes, scope.Resource.GroupResource().String(), requestmetrics.Patch)
+			body, err := limitedReadBodyWithRecordMetric(ctx, req, scope.MaxRequestBodyBytes, scope.Resource.GroupResource().String(), requestmetrics.Delete)
```

Comment:
fyi @logicalhan it looks like we were recording DELETEs as PATCHes in the metrics?

Reply:
Yeah, looks like so, that sucks.